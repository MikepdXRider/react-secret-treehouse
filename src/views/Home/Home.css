.treehouse {
  max-width: 20vw;
}

.enter {
  background: rgb(254, 240, 174);
  padding: 10px 20px;
  color: rgb(1, 114, 251) !important;
  text-decoration: none;
  border-radius: 5px;
}

.enter:hover {
  background-color: rgb(129, 210, 97);
}
