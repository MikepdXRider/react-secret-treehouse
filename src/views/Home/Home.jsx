import { Link } from 'react-router-dom';
import styles from './Home.css';
import treehouse from '../../assets/treehouse.png';

export default function Home() {
  return (
    <>
      <img src={treehouse} alt="Treehouse" className={styles.treehouse} />
      <h1>
        <Link to="/treehouse" className={styles.enter}>
          Enter Treehouse
        </Link>
      </h1>
      <Link to="/treehouse/members">
        See who's in the Treehouse
      </Link>
    </>
  );
}
