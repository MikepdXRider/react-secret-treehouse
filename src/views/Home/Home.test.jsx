import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Home from './Home';

it('should render the home page', () => {
  const { container } = render(
    <MemoryRouter>
      <Home />
    </MemoryRouter>
  );
  expect(container).toMatchSnapshot();
});
