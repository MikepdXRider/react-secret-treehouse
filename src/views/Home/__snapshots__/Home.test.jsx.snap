// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the home page 1`] = `
<div>
  <img
    alt="Treehouse"
    class="treehouse"
    src="/path/image"
  />
  <h1>
    <a
      class="enter"
      href="/treehouse"
    >
      Enter Treehouse
    </a>
  </h1>
  <a
    href="/treehouse/members"
  >
    See who's in the Treehouse
  </a>
</div>
`;
